rescript/plugin_setup.py

```diff
@@ -464,6 +466,46 @@
 )
 
 
+plugin.methods.register_function(
+    function=replace_taxonomy,
+    inputs={
+        'taxonomy': FeatureData[Taxonomy]
+    },
+    parameters={
+        'taxonomy_replacement_map': MetadataColumn[Categorical]
+    },
+    outputs=[('edited_taxonomy', FeatureData[Taxonomy])],
+    input_descriptions={
+        'taxonomy': 'Taxonomy strings data to be edited.'
+    },
+    parameter_descriptions={
+        'taxonomy_replacement_map': 'A two-column tab-delimitad file in which '
+                                    'the rank-lineage string of the first '
+                                    'column is replaced by the rank-lineage '
+                                    'string of the scond column. For  '
+                                    'example, replacing all instances of '
+                                    '\'g__Escherichia;\' with '
+                                    '\'g__Escherichia-Shigella;\'. All items '
+                                    'in the first column must be unique!'},
+    output_descriptions={
+        'edited_taxonomy': 'Taxonomy in which the original rank-lineage '
+                           'strings were replaced by user-supplied '
+                           'rank-lineage strings.'
+    },
+    name='Replaces existing taxonomy strings with those provided',
+    description=('This method allows the user to edit taxonomy strings. '
```

Comment:
I recommend trimming back the description — it is a bit too specific/applied to a specific example. I recommend using a tutorial/usage example to give this example, but it is TMI for the description

Reply:
Agreed.